data/bingo/functions/init/init.mcfunction

```diff
@@ -48,13 +48,13 @@ forceload add 0 0
 	# 	inventory. Should look like this: "clear @a[tag=bingo.clear] <item> 1
 	# categories: (String List) List of category names. All categories referenced
 	# 	here should be registered separately.
-	# weight: (Integer) Weight of this item for the rng.
+	# weight: (Integer) Weight of this item for the RNG.
 	#
 	# Categories have the following structure:
-	# name: (String) Namespaced id to uniquly identify the category within bingo
-	# translateableName: (String) Text component used for displaying the category's
+	# name: (String) Namespaced id to uniquely identify the category within bingo
+	# translatableName: (String) Text component used for displaying the category's
 	# 	name
-	# items: (Compund List) (readonly) List of all items this category has. Entries
+	# items: (Compound List) (read-only) List of all items this category has. Entries
```

Comment:
`readonly` is a typical keyword in programming and should be spelled as one word